docker-compose.prod.yml

```
@@ -34,6 +50,7 @@ services:
       APP_ENV: prod
       APP_DEBUG: 0
       APP_SECRET: EDITME
+      LOAD_FIXTURES: 1
```

```suggestion
      LOAD_FIXTURES: 0
```
Is there a reason why it is enabled on production?

---

I still want to preserve the flow where Sylius shows an excellent shop front by default. But I got feedback that it may be weird from DX perspective. That's why I kept this easy switch.

Without loaded fixtures user will get error page by default - that's blah :(

---

Hmm, I see but on the other hand, it removes all data in the database in every single build which is also not ok. As it is production, IMO we shouldn't care about the error page because the admin can configure everything including channels.
Nevertheless, it's a docker compose file that can be overwritten so I'm ok with it as it is 👍️

---

I will think about it. Like I get pros and cons and maybe I should give it a try ;p

---

I decided to give it a try :)